packages/code-du-travail-api/src/server/routes/__tests__/__snapshots__/items.test.js.snap

```diff
@@ -56,7 +56,7 @@ Object {
     Object {
       "_id": "2",
       "_index": "cdtn_document_test",
-      "_score": 22.295755,
+      "_score": 22.58624,
```

Comment:
if you don't update api, there is no need to update api score can you check you ES version (7.4.1)